apis/placement/v1beta1/clusterresourceplacement_types.go

```diff
@@ -207,6 +219,57 @@ type TopologySpreadConstraint struct {
 	WhenUnsatisfiable UnsatisfiableConstraintAction `json:"whenUnsatisfiable,omitempty"`
 }
 
+// RolloutStrategy describes how to update existing placed resource either onto a different cluster
+// or in-place upgrade to the latest resource.
```

Comment:
RolloutStrategy describes how to roll out a new change in selected resources to target clusters.